src/server/db_slice.cc

```diff
@@ -660,6 +660,19 @@ void DbSlice::PostUpdate(DbIndex db_ind, PrimeIterator it, bool existing) {
     stats->strval_memory_usage += value_heap_size;
   if (existing)
     stats->update_value_amount += value_heap_size;
+
+
+  // Check if the key is watched.
+  // TODO: eliminate ToString() call
+  auto key = it->first.ToString();
+  VLOG(0) << "post update for key: " << key;
```

Comment:
We should pass the key to `PostUpdate`. If this is not an option, then maybe store some local `std::string` to call `GetSlice()`. However GetSlice still seems to be complicated.

Reply:
1. you can easily avoid running the code in the common case if you check first whether `watched_keys_` are empty.
2. Lets pass they key to `PostUpdate`.